pycbc/workflow/matched_filter.py

```diff
@@ -240,10 +239,10 @@ def setup_matchedfltr_dax_generated_multi(workflow, science_segs, datafind_outs,
 
     if match_fltr_exe == 'pycbc_multi_inspiral':
         exe_class = select_matchedfilter_class(match_fltr_exe)
-        cp.set('inspiral', 'longitude',\
-               str(radians(float(cp.get('workflow', 'ra')))))
-        cp.set('inspiral', 'latitude',\
-               str(radians(float(cp.get('workflow', 'dec')))))
+        cp.set('inspiral', 'ra',
+               str(cp.get('workflow', 'ra')))
+        cp.set('inspiral', 'dec',
+               str(cp.get('workflow', 'dec')))
```

Comment:
Does this change imply that the config files will now have to specify RA and declination in radians, or that the inspiral executable now takes degrees?

Reply:
I believe I changed this because `pycbc_multi_inspiral` was changed to use RA and declination in radians.

Reply:
I believe I was thinking that with `pycbc_multi_inspiral` now taking RA and dec, it could be directly provided from the config file without any conversion. Looking at it again, I'm not sure if that's correct. I'll have to look into this further.

Reply:
Has this been looked into?

Reply:
On the call, we decided to keep the conversion from degrees (easy for whoever sets up the .ini file from a GRB) to radians (which is what `pycbc_multi_inspiral` understands).